pkgs/development/libraries/onnxruntime/default.nix

```diff
@@ -152,9 +189,20 @@ stdenv.mkDerivation rec {
     "-Donnxruntime_USE_FULL_PROTOBUF=OFF"
   ] ++ lib.optionals pythonSupport [
     "-Donnxruntime_ENABLE_PYTHON=ON"
+  ] ++ lib.optionals cudaSupport [
+    "-DFETCHCONTENT_SOURCE_DIR_CUTLASS=${cutlass}"
+    "-Donnxruntime_USE_CUDA=ON"
+    "-Donnxruntime_USE_NCCL=ON"
```

Comment:
Better pass the unconditional `(lib.cmakeBool "USE_CUDA" cudaSupport)` (same for nccl) outside `optionals`, so that `OFF` is also explicit when it's `OFF`

Reply:
Done, should this be done for the Python flag as well?

Reply:
If you feel like including a small refactoring in the PR. Not a necessity though because outside the scope